keymanapp/keyman
# chore(android/samples): Remove Sentry dependencies

Follow-on to #7897 and fixes #7896

The Android sample/test apps don't use Sentry, so this removes the Sentry dependency. This means consumers of Keyman Engine for Android also won't need to include Sentry in their build.

## User Testing
Setup - Install the PR builds of the specified apps on an Android device/emulator

* **TEST_KEYMAN** - 
1.  Install Keyman for Android
2. Type on the in-app keyboard
3. Verify text is output in the app
4. Long-press on the globe key to display the keyboard picker menu
5. Click the "i" icon to get info on sil_euro_latin keyboard
6. Verify the keyboard info appears **with** a QR code for sharing
 
* **TEST_KMSAMPLE1** - Verifies KMSample1 functions
1. Install KMSample1
2. Type on the in-app keyboard
3. Verify text is output in the app
4. Click a suggestion from the suggestion banner
5. Verify the suggestion is inserted into the app.
6. Long-press on the globe key to display the keyboard picker menu
7. Click the "i" icon to get info on the tamil keyboard
8. Verify the keyboard info appears **without** a QR code for sharing

* **TEST_FV** - Verifies FirstVoices app functions
1. Install FirstVoices for Android
2. On the starting menu, click "Select keyboards" and enable a FirstVoices keyboard (any region & language)
3. On the starting menu, click "Setup" and enable FirstVoices and set as the current input method
4. Launch a separate app (e.g. Chrome) and select a text field
6. Set the keyboard to FirstVoices
7. Type with the FirstVoices keyboard
8. Verify text is output
9. Long-press on the globe key to display the keyboard picker menu
10. Click the "i" to get info on the FV keyboard
11. Verify the keyboard info appears **without** a QR code for sharing.
